js/src/button.js

```diff
@@ -90,6 +90,11 @@ const Button = (($) => {
           }
 
           if (triggerChangeEvent) {
+            if (input.disabled || rootElement.disabled ||
```

Comment:
there shouldn't be a need to check if the element is actually `disabled`, as this would only be exposed for elements that actually support the `disabled` attribute (e.g. `<button>`), and those elements wouldn't trigger the `click` event anyway when clicked)